src/ui/Misc/FileTypes.ts

```diff
@@ -18,8 +18,8 @@ export class FileTypes {
     var objecttype = <string>Utils.getFieldValue(result, 'objecttype');
     var filetype = <string>Utils.getFieldValue(result, 'filetype');
 
-    // When @objecttype is File we fallback on @filetype for icons and such
-    if (Utils.isNonEmptyString(objecttype) && objecttype.toLowerCase() != 'file') {
+    // When @objecttype is File, Document, or ContentVersion we fallback on @filetype for icons and such
+    if (Utils.isNonEmptyString(objecttype) && !objecttype.match(/^(file|document|ContentVersion)$/i)) {
```

Comment:
I know there's currently no tests for FileTypes, but it would have been great to add at least one for this behaviour here, instead of relying on a comment for this to be future proof.